### ISSUE TITLE
Typo in User Guide

### ISSUE THREAD

**Issue by wrljet** [wwarthen/RomWBW#380]
v3.4.0 User Guide PDF on page 84:

```
NOTE: The FLASH utility can not determine the type of your ROM chip if it is write protected.
Additionally, it has not way to even determine if it is write protected.
```

"not way" should perhaps be "no way".

Bill

**Comment by wwarthen** [comment]
Thanks Bill.  I will fix it with my next check-in.

Happy New Year!

Wayne

**Comment by wrljet** [comment]
Hey!  Happy New Year to you, too!  
Do you ever hear from Douglas?

Bill

**Comment by wwarthen** [comment]
Well, the last communication I had with Douglas was almost 5 years ago.  He was resurrecting an N8 board which did work.  It sounded like he was pretty busy with "real" work.  😀

**Comment by wwarthen** [comment]
Corrected in commit cc4ad0d.  Thanks Bill!
